package.json

```diff
@@ -11,7 +11,10 @@
     "react": "^15.4.2",
     "react-bootstrap": "^0.30.7",
     "react-dom": "^15.4.2",
-    "react-table": "^5.3.2"
+    "react-table": "^5.3.2",
+    "body-parser": "^1.16.1",
```

Comment:
There's no technical reason we _couldn't_ have just one package.json, but we should probably have a separate package.json for API code.  It will make automated builds faster if we only have to download fresh dependencies for the application in question, and should improve organization.

As an aside, common convention (nowadays) in front end JS world is to specify dependencies (served to the browser) vs. dev-dependencies (build tools, unit test runners, linters, etc).  When you see npm package installation instructions, you'll often see
`npm install --save {packageName}`
vs.
`npm install --save-dev {packageName}`

Both commands will add the package to your package.json, but under different keys.

The FE repo doesn't have anything listed in devDependencies right now.  If you look at the "scripts" in package.json, you'll see "react-scripts start", which handles all of the build for you until you're ready to take control by "ejecting" your application.  The create-react-app docs call these "transitive dependencies".